internal/dsync/drwmutex.go

```diff
@@ -60,7 +60,7 @@ const (
 	drwMutexRefreshInterval = 10 * time.Second
 
 	// maximum time to sleep before retrying a failed blocking lock()
-	lockRetryInterval = 50 * time.Millisecond
+	lockRetryInterval = 100 * time.Millisecond
```

Comment:
Let's make this configurable? So that we can test a few values?

Reply:
Sure. I'll merge patch by @vadmeste into this.